tests/nvme_test.py

```diff
@@ -105,7 +105,18 @@ def test_ctrl_info(self):
         self.assertFalse(info.features & BlockDev.NVMEControllerFeature.SANITIZE_OVERWRITE)
         self.assertFalse(info.features & BlockDev.NVMEControllerFeature.SECURE_ERASE_ALL_NS)
         self.assertFalse(info.features & BlockDev.NVMEControllerFeature.SECURE_ERASE_CRYPTO)
+        self.assertFalse(info.features & BlockDev.NVMEControllerFeature.STORAGE_DEVICE)
+        self.assertFalse(info.features & BlockDev.NVMEControllerFeature.ENCLOSURE)
+        self.assertFalse(info.features & BlockDev.NVMEControllerFeature.MGMT_PCIE)
+        self.assertFalse(info.features & BlockDev.NVMEControllerFeature.MGMT_SMBUS)
         self.assertEqual(info.fguid, "")
+        self.assertEqual(info.pci_vendor_id, 0)
+        self.assertEqual(info.pci_subsys_vendor_id, 0)
+        self.assertIn("Linux", info.model_number)
+        self.assertGreater(len(info.serial_number), 0)
+        self.assertGreater(len(info.firmware_ver), 0)
+        self.assertGreater(len(info.nvme_ver), 0)
+        self.assertEqual(info.controller_type, BlockDev.NVMEControllerType.UNKNOWN)
```

Comment:
Failed tests suggest `BlockDev.NVMEControllerType.IO` should be here instead of `UNKNOWN`.

Reply:
Ahh, another kernel nvme target difference. Let's just remove this test.